lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -2089,4 +2072,33 @@ protected function switchPersisterContext($offset, $limit)
 
         $this->currentPersisterContext = $this->limitsHandlingContext;
     }
+
+    /**
+     * Retrieves class identifiers types
+     *
+     * @param ClassMetadata $class
+     * @return array
```

Comment:
Please document the returned type in more detail - is this a `Type[]`? `string[]`?